DYDX-Bots/Large_Borrower_Allocation/src/findings.ts

```diff
@@ -0,0 +1,19 @@
+import { LogDescription, Finding, FindingSeverity, FindingType } from "forta-agent";
+
+export const createFinding = (log: LogDescription) => {
+  return Finding.fromObject({
+    name: "Large borrower allocation change detected on dYdX perpetual exchange.",
```

Comment:
I think using `change` can be understood as a large difference between the old and new allocation, while this bot is detecting borrowers with large allocation.